banzai/frames.py

```diff
@@ -218,12 +197,21 @@ def __itruediv__(self, other):
             self.primary_hdu.__itruediv__(other)
         return self
 
+    def __getitem__(self, item):
+        return self._hdus[self._hdu_keys[item]]
+
+    def __setitem__(self, key, value):
```

Comment:
Should we enforce that whatever the HDU key is matches the Data object's `name` attribute? Or set it here if not specified?

e.g. if a user of this class did something like:

```
image['SCI'] = CCDData(data=data, meta=meta, name='BPM') 
```

we could get into a weird state where the name of the HDU doesn't match the key created